third-party/realdds/doc/DDS Use cases.md

```diff
@@ -0,0 +1,255 @@
+**Trigger** - `Intel RealSense` device is disconnected from the **server**.
+
+**Successful Result** - The **Server** and all **user**s have released the resources related to the camera.
+
+**Main Scenario**
+1. The **server** will destroy the appropriate *device-info* writer.
+2. Subscribed **user**s will be notified via DDS middleware about the writer destruction (subscription unmatched) and will release all related resources.
+3. The **server** will release all other device related resources.
+
+**Variations**
```

Comment:
I don't think it helps the user to see all these with "None" - IMO, just remove them

Reply:
Many use case documents I have seen, especially ones that comply to some safety standard, use this kind of template for use case description. The template fields are kept and marked with "None" or "N/A" to signal that they are empty and not forgotten.
@Nir-Az your thoughts on this?

Reply:
AFAIK, it's better to use a template even if we have none.
It means we explicitly say no variations are expected, if it won't be there it will be ether there is no or we forgot it..

Reply:
ok